boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu1.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: lpcxpresso55s69_cpu0
```

Comment:
I may have misunderstood something, but should this be `lpcxpresso55s69_cpu1` if the NS app is running on the second core?